apps/pie-storybook/stories/testing/pie-thumbnail.test.stories.ts

```diff
@@ -50,17 +82,29 @@ const Template: TemplateFunction<ThumbnailProps> = ({
     variant,
     src,
     alt,
+    disabled,
+    hasPadding,
+    backgroundColor,
+    placeholder,
 }) => html`
     <pie-thumbnail
         variant="${variant}"
         src="${src}"
-        alt="${alt}">
+        alt="${alt}"
+        ?disabled="${disabled}"
+        ?hasPadding="${hasPadding}"
+        backgroundColor="${backgroundColor}"
+        .placeholder="${placeholder}">
     </pie-thumbnail>`;
 
 // Define the prop options for the matrix
 const sharedPropOptions = {
-    src: ['https://www.pie.design/assets/img/jet-logo-narrow.svg'],
-    alt: ['JET logo'],
+    src: [defaultArgs.src],
+    alt: [defaultArgs.alt],
+    disabled: [defaultArgs.disabled],
+    hasPadding: [defaultArgs.hasPadding],
```

Comment:
we need to generate a visual snapshot for both when the component true and false so the values should be

 ```suggestion
    disabled: [true, false],
    hasPadding: [true, false],
```